docs/release_notes/release_notes_lading_page.md

```diff
@@ -0,0 +1,12 @@
+# Release Notes
+Our monthly release notes include new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## Release Notes 2022
```

Comment:
I was thinking that I would create a new heading for each year. Or is this heading redundant?